content/en/blog/2023/ecs-otel-semconv-convergence.md

```diff
@@ -0,0 +1,50 @@
+---
+title:
+  Announcing the Elastic Common Schema (ECS) and OpenTelemetry Semantic
+  Convention Convergence
+linkTitle: ECS and OTel SemConv Convergence
+date: 2023-04-17
+spelling: cSpell:ignore ECS Reiley SemConv Yang
+author: '[Reiley Yang](https://github.com/reyang)'
+---
+
+Today, we're very excited to make a joint announcement with
+[Elastic](https://www.elastic.co/) about the future of
+[ECS](https://www.elastic.co/guide/en/ecs/master/ecs-reference.html) (Elastic
+Common Schema) and the OpenTelemetry Semantic Conventions.
```

Comment:
```suggestion
[Elastic Common Schema](https://www.elastic.co/guide/en/ecs/master/ecs-reference.html)
(ECS) and the [OpenTelemetry Semantic Conventions][].
```

The `[OpenTelemetry Semantic Conventions]` markdown link definition is added below.

Reply:
Ok with this also

Reply:
Updated.